### PR TITLE
Update env.dist file to match the rest of webmaker.org

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

File: env.dist [hunk 1]
```diff
@@ -26,7 +17,24 @@ export MOX_PORT=12319
 export S3_DOMAIN="http://localhost:12319"
 export S3_EMULATION=false
 
+export ENABLE_GELF_LOGS=false
+
+# Where can we find the login API endpoint
+export LOGIN="http://localhost:3000"
```
Comment (humphd) [inline env.dist:26]: Is this one OK in terms of Webmaker Suite (i.e., not colliding with an existing port we already use)?

File: env.dist [hunk 2]
```diff
@@ -26,7 +17,24 @@ export MOX_PORT=12319
...
+
+# What is our fully qualified login API acces point?
+export LOGINAPI="http://testuser:password@localhost:3000"
```
Comment (humphd) [inline env.dist]: I don't think we need to talk to Login at all, we just force you to be logged in already.

File: env.dist [hunk 3]
```diff
@@ -26,7 +17,24 @@ export MOX_PORT=12319
...
+
+# Persona locations - UNCOMMENT FOR USE WITH LOCAL PERSONA
```
Comment (humphd) [inline env.dist]: Same for Persona--we won't have front-end for this.

File: env.dist [hunk 4]
```diff
@@ -26,7 +17,24 @@ export MOX_PORT=12319
...
+
+# And a cookie secret
+export SESSION_SECRET="dummy secret value"
+
+# A domain for webmaker super cookies
+# You should only set this when running in a non-localhost environment
+export COOKIE_DOMAIN=""
+
 # Is this server running behind an SSL-enabled load-balancer?
 export FORCE_SSL=false
```
Comment (humphd) [inline env.dist:36]: This is just for testing/dev, I guess?  In prod you'd be on SSL?

File: env.dist (hunk 1 above)
Comment (alicoding) [inline env.dist:26]: I'm not sure about this one? I thought login suppose to be the same for all apps? or are you referring to https://github.com/humphd/makedrive/pull/2/files#diff-45473cb0daa25d5231f105558b5b7226L8

File: env.dist (hunk 4 above)
Comment (alicoding) [inline env.dist:36]: I think yes?

File: env.dist (hunk 1 above)
Comment (humphd) [inline env.dist:26]: Yeah, you're right.  I misread this as the port we'd use for this app.

**Pull request merged**